### PR TITLE
Basic tests fix base

### PR DESCRIPTION
**Pull request by rgbkrk**
This fixes a regression I put in while also adding new tests to make sure it doesn't happen again. 😉

### CODE REVIEW

File: vdom/tests/test_core.py [hunk 1]
```diff
@@ -18,25 +19,71 @@ def test_flatten_children():
...
+            'tagName': 'h1',
+            'children': 'Our Incredibly Declarative Example',
+            'attributes': {}
+        },
+        'attributes': {}
+    }
+
+    assert toJSON(
+        div(
+            h1('Our Incredibly Declarative Example'),
```
Comment (willingc) [inline vdom/tests/test_core.py]: :-)

**Comment by rgbkrk** [comment]
> One small suggestion. Going to merge and we can iterate in time.

I think maybe it didn't post. I only see your :-) comment.

**Comment by willingc** [comment]
Oh it was just a suggestion to use `Hey nteractians.` or `Hey listen` instead of just `Hey`.

**Comment by rgbkrk** [comment]
Good idea. 😉

**Pull request merged**